### PR TITLE
Add additional headers

### PR DESCRIPTION
**Pull request by jasonpincin**
Added additional "best practice" headers on the proxied request:

* `X-Forwarded-Proto`
* `X-Real-IP` 
* `X-Forwarded-For`

Also removed some redundancy within the config to further simplify and streamline it.

### CODE REVIEW

File: etc/containerpilot.json [hunk 1]
```diff
@@ -20,11 +20,19 @@
...
-      "name": "nginx-public-ssl",
+      "name": "nginx-ssl",
       "port": 443,
       "health": "/usr/local/bin/acme init && /usr/bin/curl --insecure --fail --silent --show-error --output /dev/null --header \"HOST: {{ .ACME_DOMAIN }}\" https://localhost/nginx-health",
       "poll": 10,
       "ttl": 25,
+      "interfaces": ["eth0"]
+    },
+    {
+      "name": "nginx-public-ssl",
```
Comment (tgross) [inline etc/containerpilot.json]: I'm not sure I understand this bit: why do we have two health checks of the same port?

Comment (jasonpincin) [inline etc/containerpilot.json]: One test is on the private interface, one is on the public interface.

Comment (tgross) [inline etc/containerpilot.json]: Gotcha, and I'm seeing that I also missed that one does the `acme init` too.

**Comment by tgross** [comment]
For some reason we never merged this. Any reason not to @jasonpincin @misterbisson?

**Comment by jasonpincin** [comment]
No reason other than maybe I didn't see a LGTM?

**Comment by misterbisson** [comment]
🏡 🚶

**Pull request merged**